templates/Segment_Image_Data.ipynb

```diff
@@ -9,18 +9,17 @@
   },
```

Comment:
<p>deepcell_output_dir and label_dir are the same, we should pick a single variable name and stick with it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>